### PR TITLE
Prune stuck inprogress

### PR DESCRIPTION
**Pull request by lotharschulz**
_in-progress_ left over files from past stuck backups may still exists on the backup folder/volume.
Whenever a new container starts up left over _in-progress_ files shall be removed.

@jmcs pls check

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +0,0 @@
-scm-source.json
```
Comment (jmcs) [inline .gitignore:1]: Why did you remove the gitignore?

Comment (lotharschulz) [inline .gitignore:1]: .gitignore is not in branch _prune-stuck-inprogress_ anymore https://github.com/zalando/ghe-backup/tree/prune-stuck-inprogress

**Comment by jmcs** [comment]
The PR looks ok but there are conflicts that prevent the merge.
